src/main/scala/org/constellation/util/HealthChecker.scala

```diff
@@ -91,18 +140,23 @@ class HealthChecker[F[_]: Concurrent: Logger](
   downloader: DownloadProcess
 ) extends StrictLogging {
 
+  val recentSync: SingleRef[F, RecentSync] = SingleRef(RecentSync(Snapshot.snapshotZeroHash, 0))
```

Comment:
I don't understand why you need to keep this state as recent state is available at `SnapshotService` and provided during the check, information `recentSync` shouldn't influence the logic